recipes/watchdog-wms/meta.yaml

```diff
@@ -22,12 +23,12 @@ package:
     version: {{ version }}
 
 build:
-  number: 0
+  number: 1
```

Comment:
If you change the version number of the tool you don't need to increase the build number.